app/views/questions/show.html.erb

```diff
@@ -1,19 +1,25 @@
-<div class="col-md-4 offset-4">
-  <div class="text-center">
-    <div ><Strong><%= @question.title %></Strong></div>
-    <div><%= @question.content %></div>
+<div class=" question ">
+  <%= render partial: 'shared/error_messages', locals: { model: @question}%>
+  <div class="title"><%= @question.title %></div>
+  <div class="desc"><%= @question.content.to_plain_text %></div>
   <% if @question.pdf_attachment.present?%>
+    <%= link_to  "Open PDF",rails_blob_path(@question.pdf_attachment), class: "d-block" %>
     <iframe src="<%= rails_blob_path(@question.pdf_attachment) %>" scrolling="no" style="overflow:hidden" height="200" width="300"></iframe>
-    <%= link_to  "Open PDF",rails_blob_path(@question.pdf_attachment) %>
-    <% end %>
-    <br />
-    <div>Question topics</div>
-    <div class="topics-container">
-      <%= render partial: 'topics/topic', collection: @question.topics %>
-    </div>
-    <% if @question.user_id == current_user&.id %>
+  <% end %>
+  <br />
+  <div>Question topics</div>
+  <div class="topics-container">
+    <%= render partial: 'topics/topic', collection: @question.topics %>
+  </div>
+</div>
+<div class="question mt-2">
+  <% if @question.user_id == current_user&.id %>
     <%= button_to 'Edit Question', edit_question_path, method: :get, class: "mt-2 btn btn-primary" %>
     <%= button_to "Delete Question", @question,method: :delete, class: "mt-2 btn btn-primary", form: { data: {turbo_confirm: 'Are you sure?'}}  %>
+  <% else %>
+    <% if logged_in? %>
+      <%= render partial: 'answers/form',locals:{ entity: @question, answer:@question.answers.build } %>
     <% end %>
-  </div>
+  <% end %>
 </div>
+<%= render partial: 'answers/answer', collection: @question.answers.where.not(id: nil),locals:{ entity: @question}%>
```

Comment:
@question.answers.where.not(id: nil) ? 
what's this?

Reply:
discuss